add 2024 kcseu scaffold page (#461)
kubernetes/contributor-site: content/en/events/2024/kcseu/faq.md

---
title: "Frequently Asked Questions (FAQ)"
type: docs
weight: 7
---

- [When is the summit taking place?](#when-is-the-summit-taking-place)
- [Where is the summit located?](#where-is-the-summit-located)
- [Will I be able to attend remotely?](#will-i-be-able-to-attend-remotely)
- [What are the Health and Safety protocols](#what-are-the-health-and-safety-protocols)
- [Why do I need to be a Kubernetes Org member to attend in-person?](#why-do-i-need-to-be-a-kubernetes-org-member-to-attend-in-person)
- [What is a sponsored attendee?](#what-is-a-sponsored-attendee)
- [I have an idea for a session](#i-have-an-idea-for-a-session)
- [My SIG wants to meet at the Summit](#my-sig-wants-to-meet-at-the-summit)
- [Will there be a doc sprint?](#will-there-be-a-doc-sprint)
- [Will there be a New Contributor Workshop?](#will-there-be-a-new-contributor-workshop)
- [Can I bring a guest to the Social?](#can-i-bring-a-guest-to-the-social)
- [I have a question! How do I contact the event staff?](#i-have-a-question-how-do-i-contact-the-event-staff)

### When is the summit taking place?

> TBD

Reviewer: 19.04.2024

### Where is the summit located?

1 Place de la Porte de Versailles, 75015 Paris, France

### Will I be able to attend remotely?

> No, based on our experience with Kubecon NA 2021, we do not expect to be able

Reviewer 2: ```suggestion
No, based on our experience with Kubecon EU 2023, we do not expect to be able
```

Health & Safety Guidelines</a>. There may be additional restrictions that
develop closer to the event, and we will update the event accordingly to follow
venue, municipality and CDC guidelines at a minimum.

### Why do I need to be a Kubernetes Org member to attend in-person?

We welcome everyone to start contributing to the project, but with 1600+
Kubernetes Org members and limited space and budget requirements we've
made the decision to limit attendance to those that are already actively
engaged with the project.

If you are not yet a member, but active within the project, please consider
joining! The process of joining is straight forward. See our [org membership]
docs for more information. If your application to join is pending, please
contact us at community@kubernetes.io about registering.

Kubernetes Orgs:

<ul>
<li><a href="https://github.com/kubernetes" rel="noopener noreferrer" target="_blank">kubernetes</a></li>
<li><a href="https://github.com/kubernetes-client" rel="noopener noreferrer" target="_blank">kubernetes-client</a></li>
<li><a href="https://github.com/kubernetes-csi rel="noopener noreferrer" target="_blank">kubernetes-csi</a></li>

> <li><a href="https://github.com/kubernetes-sigs" rel="noopener noreferrer" target="_blank">kubernetes-sigs</a></li>
> </ul>
> 

Reviewer 3: ```suggestion
<li><a href="https://github.com/kubernetes-sigs" rel="noopener noreferrer" target="_blank">kubernetes-sigs</a></li>
<li><a href=“https://github.com/etcd-io” rel=“noopener noreferrer” target=“_blank”>etcd-io</a></li>
</ul>

```

etcd-io is now a k8s org

[org membership]: https://github.com/kubernetes/community/blob/master/community-membership.md#member

### What is a sponsored attendee?

If you are a SIG lead or subproject owner and there is someone that is not yet
an org member or from another project that might take part in a panel,
SIG face-to-face meeting etc, you can sponsor them to attend the summit.
Please [email us] to make arrangements.

### I have an idea for a session

Most of the Summit will be unconference-style, so you will have an opportunity
to propose a session closer to the Summit itself. If you have an idea for a talk,
workshop or panel please post your idea to the unconference issue or the CFP
form once they become available.  

### My SIG wants to meet at the Summit

> We do not know yet if we will be able to request additional space outside of
> the colo event space on Tuesday, April 18th. We will update this page and send
> out community wide messages about it once the details are available.

Reviewer 2: ```suggestion
We plan to include SIG meetings as part of the schedule for the Summit.  Please watch for the CfP, and submit your proposed SIG meeting as a Summit session.
```

full day from 11am to 5pm.

### Will there be a New Contributor Workshop?

No, they are very time, person and resource intensive. We encourage those
interested in becoming a contributor to use the New Contributor Course 
slated to launch later this year.

Until then, here are some additional resources you may find useful:
- [New Contributor Workshop 2019 - Barcelona](https://www.youtube.com/watch?v=BQ7y2TFOzF4&list=PL69nYSiGNLP2WTJ6P8sQenhf0RY-JqF5L)
- [New Contributor Workshop 2019 - San Diego](https://www.youtube.com/watch?v=uUJrGwAom-E&list=PL69nYSiGNLP0OWp38tPBc-jSlMmwWr6Ci&index=15)
- [Kubernetes Contributor Guide](/docs/guide/)

### Can I bring a guest to the Social?

Contributors will be allowed to bring a single family member
or partner to the social (a "plus one"). 
[See the Social page to request a guest pass](https://www.kubernetes.dev/events/2023/kcseu/social/).

### I have a question! How do I contact the event staff?

You can [email us] or pop into the
<a href="https://kubernetes.slack.com/messages/contributor-summit" rel="noopener noreferrer" target="_blank">
#contributor-summit</a> Slack channel to ask a question. :)

> [email us]: mailto:community@kubernetes.io

Reviewer 3: ```suggestion
[email us]: mailto:summit-team@kubernetes.io
```
